tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,677 @@
+ var center = simple.centroid();
+ // Create buffer around geometry; the number represents the width of buffer in meters
+ var buff = center.buffer(5000);
+ // return variable from function
+ // Change the returned variable to visualize what each sequential geometry operation does
+ return buff;
+}
+// Map function over feature collection
+var mappedCentroid = countyConnect.map(performMap);
+// Add the layer to the map with a specified color and layer name
+Map.addLayer(mappedCentroid, {color: 'blue'}, 'Mapped buffed centroids');
+```
+
+---
+## Operations on features
+
+### Set property name and value of geometry to create a feature
+```javascript
+var feat = ee.Feature(geometry, {Name: 'featureName', Size: 500});
+```
+### Create a new feature from existing feature while renaming a property
+```javascript
+var featNew = feature.select(['name'], ['descriptor']);
+```
+### Extract values of a property from a Feature
+```javascript
+var featVal = feature.get('size');
+```
+
+#### Feature operations example
+
+- Let's create a feature from scratch and play around with its properties
+
+```javascript
+// Create geometry
+var varGeometry = ee.Geometry.Polygon(0, 0, 40, 30, 20, 20, 0, 0);
+// Create feature from geometry
+var varFeature = ee.Feature(varGeometry, {
+ Name: ['Feature name', 'Supreme'],
+ Size: [500, 1000]
+});
+// Get values of a property
+var arr=varFeature.get('Size');
+// Print variable
+print(arr);
+// Select a subset of properties and rename them
+var varFeaturenew = varFeature.select(['Name'], ['Descriptor']);
+// Print variable
+print(varFeaturenew);
+```
+
+---
+
+## Filters
+
+### Creator a filter for values of a property
+
+```javascript
+var bFilter = ee.Filter.eq(propertyName, value);
+```
+>or .neq , .gt , .gte , .lt , and .lte
+
+### Create a filter based on maximum difference from a threshold
+
+```javascript
+var diffFilter = ee.Filter.maxDifference(threshold, propertyName, value);
+```
+
+### Create a text filter
+
+```javascript
+var txtFilter = ee.Filter.stringContains(propertyName, stringValue);
+```
+
+>or .stringStartsWith, and .stringEndsWith
+
+### Create a range filter
+
+```javascript
+var rangeFilter = ee.Filter.rangeContains(propertyName, stringValue, minValue, maxValue);
+```
+  
+### Create a list filter to check for certain values
+
+```javascript
+var listFilter = ee.Filter.listContains(propertyName, value1, propertyName2, value2);
+```
+>.inList to test against list of values
+
+### Create a filter of dates
+
+```javascript
+var dateFilter = ee.Filter.calendarRange(startDate, stopDate);
+```
+
+### Create a filter for particular days of the year
+
+```javascript
+var dayFilter = ee.Filter.dayOfYear(startDay, stopDay);
+```
+
+### Create a filter to subset geospatial data
+
+```javascript
+var boundsFilter = ee.Filter.bounds(geometryOrFeature);
+```
+### Combining and inversing filters
+
+```javascript
+var newFilter = ee.Filter.and(listOfFilters);
+var newFilter = ee.Filter.or(listOfFilters);
```

Comment:
Please `newFilter` in these two lines so that the same variable isn't declared twice.